src/components/atoms/SvgLoop/SvgLoop.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useEffect, useRef, useState } from "react";
+import classNames from "classnames";
+
+export interface SvgLoopProps
+  extends React.ImgHTMLAttributes<HTMLImageElement> {
+  sources: string[];
+  delay?: number;
+}
+
+const pause: (delay: number) => Promise<void> = (delay) =>
+  new Promise((resolve) => setTimeout(resolve, delay));
+
+export const SvgLoop: React.FC<SvgLoopProps> = ({
```

Comment:
A component that acts like a looping GIF when given several sources. Advantage is it may loop over SVG files which are scalable without losing quality.